platform/default/include/mbgl/gfx/headless_backend.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <mbgl/gfx/renderable.hpp>
+#include <mbgl/gfx/renderer_backend.hpp>
+#include <mbgl/util/image.hpp>
+
+#include <memory>
+
+namespace mbgl {
+namespace gfx {
+
+class HeadlessBackend : public gfx::Renderable {
```

Comment:
nit: a short class description would be nice

Reply:
Done.